src/AndroidDebugLauncher/Launcher.cs

```diff
@@ -244,10 +244,18 @@ private LocalLaunchOptions SetupForDebuggingWorker(CancellationToken token)
                                 allowedAbis = new DeviceAbi[] { DeviceAbi.armeabi, DeviceAbi.armeabiv7a };
                                 break;
 
+                            case TargetArchitecture.ARM64:
+                                allowedAbis = new DeviceAbi[] { DeviceAbi.arm64v8a };
+                                break;
+
                             case TargetArchitecture.X86:
                                 allowedAbis = new DeviceAbi[] { DeviceAbi.x86 };
```

Comment:
I originally had this, but a 64 bit should report both abi's as compatible. My thinking was that if a device supported ONLY the 64 bit abi, then it would actually be incompatible with 32 bit. What do you think?